artifact/v3/manifest/manifest.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <artifact/v3/manifest/manifest.hpp>
+
+#include <string>
+#include <sstream>
+
+#include <common/error.hpp>
+
+namespace artifact {
+namespace v3 {
+namespace manifest {
```

Comment:
I probably missed this earlier, but should the first namespace be `mender`?

Reply:
Probs right.

Reply:
so, just to be sure: `mender::artifact::v3::manifest` ?

Reply:
Yes. I didn't want to type it out in case you thought I meant to change it to the non C++11 style. 🙂